app/assets/javascripts/controllers/security_group/security_group_form_controller.js

```diff
@@ -20,33 +18,27 @@ ManageIQ.angular.app.controller('securityGroupFormController', ['securityGroupFo
     vm.newRecord = securityGroupFormId === "new";
     vm.saveable = miqService.saveable;
 
-    if (vm.newRecord) {
-      vm.afterGet = true;
-      vm.modelCopy = angular.copy( vm.securityGroupModel );
-    } else {
-      miqService.sparkleOn();
-      API.get("/api/security_groups/" + securityGroupFormId + "?attributes=name,ext_management_system.name,description,cloud_tenant.name,firewall_rules").then(function(data) {
-        vm.securityGroupModel.name = data.name;
-        vm.securityGroupModel.ems_name = data.ext_management_system.name;
-        vm.securityGroupModel.description = data.description;
-        vm.securityGroupModel.cloud_tenant_name = angular.isDefined(data.cloud_tenant) ? data.cloud_tenant.name : undefined;
-        vm.securityGroupModel.firewall_rules = data.firewall_rules;
-        vm.securityGroupModel.firewall_rules_delete = false;
-        vm.afterGet = true;
-        vm.modelCopy = angular.copy( vm.securityGroupModel );
-      }).catch(miqService.handleFailure);
-      miqService.sparkleOff();
-    }
-  };
-
-  var getSecurityGroups = function() {
     API.get("/api/security_groups/?expand=resources&attributes=ems_ref,id,name").then(function(data) {
       vm.security_groups_list = data.resources;
+    }).then(function() {
+      if (vm.newRecord) {
+        vm.afterGet = true;
+        vm.modelCopy = angular.copy( vm.securityGroupModel );
+      } else {
+        miqService.sparkleOn();
+        API.get("/api/security_groups/" + securityGroupFormId + "?attributes=name,ext_management_system.name,description,cloud_tenant.name,firewall_rules").then(function(data) {
+          Object.assign(vm.securityGroupModel, data);
+          vm.securityGroupModel.firewall_rules_delete = false;        vm.securityGroupModel.name = data.name;
```

Comment:
why that `vm.securityGroupModel.name = data.name;` ? Isn't that handled by the assign too? (But if not, formatting..)

Reply:
Yes, that's a residual cut mistake, removed, thanks!